doc/nrf/releases_and_maturity/migration/migration_guide_2.9.0-nrf54h20.rst

```diff
@@ -0,0 +1,205 @@
+
+nrfutil
++++++++
+
+.. toggle::
+
+  * ``nrfutil`` has been updated to version 7.13.0.
+
+    Install nRF Util version 7.13.0 as follows:
+
+      1. Download the nRF Util executable file from the `nRF Util development tool`_ product page.
+      #. Add nRF Util to the system path on Linux and MacOS, or environment variables on Windows, to run it from anywhere on the system.
+         On Linux and MacOS, use one of the following options:
+
+         * Add nRF Util's directory to the system path.
+         * Move the file to a directory in the system path.
+
+      #. On MacOS and Linux, give ``nrfutil`` execute permissions by typing ``chmod +x nrfutil`` in a terminal or using a file browser.
+         This is typically a checkbox found under file properties.
+      #. On MacOS, to run the nRF Util executable, you need to allow it in the system settings.
+      #. Verify the version of the nRF Util installation on your machine by running the following command::
+
+            nrfutil --version
+
+      #. If your version is below 7.13.0, run the following command to update nRF Util::
+
+            nrfutil self-upgrade
+
+         For more information, see the `nRF Util`_ documentation.
+
+nrfutil device
+++++++++++++++
+
+.. toggle::
+
+  * ``nrfutil device`` has been updated to version 2.7.8.
+
+    Install the nRF Util ``device`` command version 2.7.8 as follows::
+
+       nrfutil install device=2.7.8 --force
+
+    For more information, consult the `nRF Util`_ documentation.
+
+nrfutil trace
++++++++++++++
+
+.. toggle::
+
+  * ``nrfutil trace`` has been updated to version 3.10.0.
+
+    Install the nRF Util ``trace`` command version 3.10.0 as follows::
+
+       nrfutil install trace=3.10.0 --force
+
+    For more information, consult the `nRF Util`_ documentation.
+
+nrfutil suit
+++++++++++++
+
+.. toggle::
+
+  * ``nrfutil suit`` has been updated to version 0.9.0.
+
+    Install the nRF Util ``suit`` command version 0.9.0 as follows::
+
+       nrfutil install suit=0.9.0 --force
+
+    For more information, consult the `nRF Util`_ documentation.
+
+Samples and applications
+========================
+
+This section describes the changes related to samples and applications.
+
+|no_changes_yet_note|
+
+.. _migration_2.9.0-nrf54h20_required_nrf54h:
+
+nRF54H20
+--------
+
+|no_changes_yet_note|
+
+Libraries
+=========
+
+This section describes the changes related to libraries.
+
+|no_changes_yet_note|
+
+.. _migration_2.9.0-nrf54h20_recommended:
+
+Recommended changes
+*******************
+
+The following changes are recommended for your application to work optimally after the migration.
+
+Samples and applications
+========================
+
+This section describes the changes related to samples and applications.
+
+Matter
+------
+
+This section describes the changes related to Matter support.
+
+|no_changes_yet_note|
+
+Libraries
+=========
+
+This section describes the changes related to libraries.
+
+|no_changes_yet_note|
```

Comment:
Just a reminder to remove these entries, may be towards the end of this release? as this a nRF54H20 migration guide.